README.md

```diff
@@ -208,6 +232,61 @@ apps:
     dockerfile: ../some_other/Dockerfile
 ```
 
+## Workflow
```

Comment:
By the way, this flow is already mostly documented in https://github.com/shakacode/heroku-to-control-plane/blob/main/docs/migrating.md#clone-the-staging-environment. Do we want to duplicate it here?

Reply:
The migration should probably refer to this.

We need to let people know about the simple flow without a full migration.